roles/openshift_node_group/files/sync.yaml

```diff
@@ -124,35 +118,26 @@ spec:
                   echo "nodeName: $KUBELET_HOSTNAME_OVERRIDE" >> /etc/origin/node/tmp/node-config.yaml
             fi
 
-            # detect whether the node-config.yaml has changed, and if so trigger a restart of the kubelet.
+            # detect whether the node-config.yaml or volume-config.yaml has changed, and if so trigger a restart of the kubelet.
             if [[ ! -f /etc/origin/node/node-config.yaml ]]; then
               cat /dev/null > /tmp/.old
             fi
-
-            if [[ ! -f /etc/origin/node/volume-config.yaml ]]; then
-              cat /dev/null > /tmp/.old-volume-config
-            fi
-
-            md5sum /etc/origin/node/tmp/node-config.yaml > /tmp/.new
-
-            if [[ ! -f /etc/origin/node/tmp/volume-config.yaml ]]; then
-              cat /dev/null > /tmp/.new-volume-config
+            
+            tmp_path=/etc/origin/node/tmp
+            if [[ -f ${tmp_path}/volume-config.yaml ]]; then
```

Comment:
It would be best to tar node-config and use `tar --append` to append volume-config if it exists